input_validation/module_utils/common/en_us_validation_msg.py

```diff
@@ -0,0 +1,136 @@
+# Copyright 2025 Dell Inc. or its subsidiaries. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+### All of these messages are used in logical_validation.py
+
+# roles_config.yml
+max_number_of_roles_msg = "a max of 100 roles can be supported."
+min_number_of_groups_msg = "at least 1 group is required."
+min_number_of_roles_msg = "at least 1 role is required."
+max_number_of_roles_per_group_msg = "Groups can support a maximum of 5 roles."
+resource_mgr_id_msg = "The resource_mgr_id needs to be set if and only if the group belongs to k8worker or slurmworker roles."
+grp_exist_msg = "A valid group must be provided."
+invalid_switch_ip_msg = "Please provide a valid switch IPv4 address (example: 10.5.0.1)."
+grp_role_msg = "Please associate this group with a role."
+parent_service_node_msg = "Group is associated with login, compiler, service, k8head, slurmhead role(s)."
+parent_service_role_dne_msg = "A service role must be present when the parent is provided."
+parent_service_role_msg = "A service role does not exist, the parent should be empty for any group associated with worker or default roles."
+bmc_static_range_invalid_msg = "Static range should be in the following format: IPv4Start-IPv4End (example: 10.5.0.1-10.5.0.200)."
+overlapping_static_range = "bmc_details static_range is overlapping with other static ranges."
+duplicate_switch_ip_msg = "Duplicate switch IP provided."
+switch_details_incomplete_msg = "If providing switch details, please provide both the IP and Ports fields."
+switch_details_no_bmc_details_msg = "If switch details are provided then bmc_details' static_range must also be provided."
+
+# provision_config.yml 
+default_lease_time_fail_msg = "Please provide a valid default_lease_time."
+timezone_fail_msg = "Unsupported Timezone. Please check the timezone.txt file for a list of valid timezones."
+enable_switch_based_fail_msg = "enable_switch_based must be set to either true or false."
```

Comment:
iso_file_path variable is moved from provision_config.yml to software_config.json. nodename will be removed. switch details moved to roles_config.yml